ConsultFlow/ConsultFlow.Core.Domain/Constants/IdentityProviders.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsultFlow.Core.Domain.Constants
+{
+    public class IdentityProviders
```

Comment:
@fwcheesecake move to namespace `ConsultFlow.Core.Domain.Entities.IdentityProviders`